inc/pkthlp.h

```diff
@@ -49,6 +49,10 @@ typedef union {
 #define QUIC_MAX_HEADER_LEN 47 // Based on RFC 9000 (QUIC v1)
 #define UDP_HEADER_STORAGE UDP_HEADER_BACKFILL(AF_INET6)
 #define TCP_HEADER_STORAGE (TCP_HEADER_BACKFILL(AF_INET6) + TCP_MAX_OPTION_LEN)
+#define ICMP4_ECHOREPLY_TYPE 0
```

Comment:
FYI the ICMP type defines are found in ipmib.h (ICMP6_ECHO_REPLY, ICMP4_ECHO_REPLY). No ICMP codes are defined there though